Library/Homebrew/dependable.rb

```diff
@@ -1,7 +1,7 @@
 require "options"
 
 module Dependable
-  RESERVED_TAGS = [:build, :optional, :recommended, :run, :linked]
+  RESERVED_TAGS = [:build, :optional, :recommended, :run]
```

Comment:
There's a bunch in `homebrew/science` as well. That's the only other tap I can see this syntax has leaked into. Hard to check for external taps because we gave it such a generic name, albeit entirely understandably.